<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raio-X Total v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>raios-x.png</normaloff>raios-x.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(158, 158, 158)</string>
  </property>
  <property name="iconSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 130, 130) ;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>70</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border: 2px solid rgb(63, 66, 72);
border-radius: 20px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo_cot1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>268</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit{
	color: rgb(255, 255, 255);
	border: none;
	background-color: transparent;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>RAIO-X TOTAL</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>511</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit{
	color: rgb(255, 255, 255);
	border: none;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Versão: 1.0</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_4">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>270</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit{
	color: rgb(235, 28, 36);
	border: none;
	background-color: transparent;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>RAIO-X TOTAL</string>
     </property>
    </widget>
    <zorder>plainTextEdit_4</zorder>
    <zorder>label_2</zorder>
    <zorder>plainTextEdit</zorder>
    <zorder>plainTextEdit_2</zorder>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>73</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color:rgb(130, 130, 130);
	border-radius: 5px;
	color: rgb(255, 255, 255);
	padding-left: 10px;
}

QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus{
	border: 2px solid  rgb(240, 71, 71);
	background-color: rgb(43, 45, 56);
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Informar Registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>73</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 2px solid rgb(63, 66, 72);
	background-color:rgb(130, 130, 130);
	border: none;
	border-radius: 5px;
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	background-color: rgb(240, 71, 71);
}

QPushButton:pressed{
	background-color:rgb(209, 61, 61);
}</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buscar1.png</normaloff>buscar1.png</iconset>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_3">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>75</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
	color: rgb(255, 255, 255);
	border: none;
	padding-top: 1px;
	padding-left: 3px;
	background-color: transparent;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Registro do Paciente: </string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>465</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
	border: none;
	color: rgb(235, 28, 36);
	background-color: transparent;
	font: bold;
}</string>
    </property>
    <property name="plainText">
     <string>Para visualizar as imagens do paciente, realizar duplo-clique em algum dos arquivos da lista.
Qualquer dúvida, favor entrar em contato com o suporte.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>145</y>
      <width>101</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	border-radius: 5px;
	background-color: rgb(130, 130, 130);
	padding-top: 5px;
	padding-left: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>117</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
	color: rgb(255, 255, 255);
	border-radius: 5px;
	background-color: transparent;
	padding-top: 3px;
	padding-left: 1px;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Data</string>
    </property>
   </widget>
   <zorder>plainTextEdit_5</zorder>
   <zorder>frame</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>plainTextEdit_3</zorder>
   <zorder>plainTextEdit_8</zorder>
   <zorder>listWidget_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
